### PR TITLE
Bah 18512

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/18512

As a developer, I need to deprecate the old VA constant value "AVGBAH" and update the application to use the new constant value "AVGVABAH" so that the constants file follows a consistent naming convention and does not lead to confusion.

## Testing done
Dev and QA tested.

## Acceptance criteria
- [ ] The new constant "AVGVABAH" is saved in the "CalculatorConstant.csv" on staging with a value
of 1681.
- [ ] The Compare Tool uses the "AVGVABAH" rate instead of the "AVGBAH" rate:
   -On the "Search Results" Page
   -On the "School Profile" Page
- [ ] The old constant "AVGBAH" is removed from the "CalculatorConstant.csv" file.
- [ ] The new constant/logic is behind a production flag.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/reducers/calculator.js [hunk 1]
```diff
@@ -310,7 +310,7 @@ export default function(state = INITIAL_STATE, action) {
         } else {
           giBillBenefit =
             action.payload.AVGDODBAH &&
-            action.payload.AVGDODBAH < action.payload.AVGBAH
+            action.payload.AVGDODBAH < action.payload.AVGVABAH
```
Comment (rianfowler) [inline src/applications/gi/reducers/calculator.js]: it doesn't look like this behavior is gated- we have a `calculator.old.js` for the selector but not the reducer. 

`FETCH_PROFILE_SUCCEEDED` only adds `AVGVABAH` on staging

Comment (hughes-dustin) [reply src/applications/gi/reducers/calculator.js]: If you look at line 301 you will see the production check that wraps this code.
`if (!environment.isProduction())`

If need be, I can create a new file and move the production flag check into `reducers/index.js`.

**Review by rianfowler — APPROVED** [review]

File: src/applications/gi/reducers/calculator.js (hunk 1 above)
Comment (rianfowler) [reply src/applications/gi/reducers/calculator.js]: I should have look further up. Thanks for pointing that out.

**Pull request merged**